tests/modules/msk/gbcms/main.nf

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { GBCMS } from '../../../../modules/msk/gbcms/main.nf'
+
+workflow test_gbcms {
+    
+    input = [
+        [ id:'test' ], // meta map
+        [],
+        [],
+        [],
+        [],
+        [],
+        params.sample,
+        // customize test specific parameters here
+        "variant_file.maf",
+        [args: '--omaf']
```

Comment:
i'm surprised the test doesn't fail, but when running the test locally i notice the following:
```
WARN: Input tuple does not match input set cardinality declared by process `test_gbcms:GBCMS` -- offending value: [[id:test], [], [], [], [], [], C-9M47TU-L001-d, variant_file.maf, [args:--omaf]]
```
Also i think the order of these inputs does not match the order in the process definition

Reply:
so apparently we get nextflow to catch this with `nextflow.enable.strict=true` or using the `NXF_ENABLE_STRICT` environmental variable.